components/file_system/src/metrics_manager.rs

```diff
@@ -27,6 +30,7 @@ impl BytesFetcher {
                 write: stats.fetch(io_type, IOOp::Write) as u64,
             },
             BytesFetcher::FromIOSnooper() => fetch_io_bytes(io_type),
+            BytesFetcher::FromProc() => thread_io::fetch_thread_io_bytes(io_type),
```

Comment:
Should be `fetch_all_thread_io_bytes`.

Reply:
But here requires `IOBytes` as return value. What shold `fetch_all_thread_io_bytes` return. Current thread IOBytes?

Reply:
This is the metrics flusher, meaning it should report the server IO flow statistics, not only a thread. And given the overhead of iterating thread locals, maybe you can change this function to `fn fetch(&self) -> [IOBytes; IOType::COUNT]`

Reply:
I added a new global array to record total io bytes. I think thats a better solution